src/cmd/ship.go

```diff
@@ -16,10 +16,11 @@ import (
 )
 
 type shipConfig struct {
-	BranchToShip  string
-	InitialBranch string
+	BranchToShip  string // the name of the branch to ship
```

Comment:
This comment doesn't seem to add anything to me and feels like it is just repeating the variable name

Reply:
Okay, removing the comment. I added it mostly for consistency since the other fields need comments more, but you are right that there isn't much to comment here.